routes/web.php

```diff
@@ -10,11 +10,25 @@
 | contains the "web" middleware group. Now create something great!
 |
 */
-
-Route::get('/', function () {
-    return view('welcome');
+	
+Route::get('/demo', function () {
+   return view('demo');
 });
+Route::get('resources/{tags?}', 'ResourcesController@index');
 
 Auth::routes();
 
+Route::get('resources/show/{id}', 'ResourcesController@show');
+
+Route::get('/demo/editModal', 'ResourcesController@getModalForm')->name('editModal');
```

Comment:
You don't need this route. You should put the modal within the HTML and use `(selector).modal('show')`